desktop/cmp/tab/TabSwitcher.js

```diff
@@ -32,19 +34,31 @@ export const [TabSwitcher, tabSwitcher] = hoistCmp.withFactory({
         const orientation = withDefault(props.orientation, 'top'),
             vertical = ['left', 'right'].includes(orientation);
 
+        const items = tabs.map(tab => {
+            const {id, title, icon, disabled, showRemoveAction, excludeFromSwitcher} = tab;
+            if (excludeFromSwitcher) return null;
+            return blueprintTab({
+                id,
+                disabled,
+                items: [
+                    icon,
+                    title,
+                    button({
+                        icon: Icon.x(),
+                        style: {minHeight: 15, minWidth: 15, borderRadius: 20, padding: 0},
```

Comment:
Should we move these styles to CSS with a class name? Also, I think we need a (small) left-margin on the button to give a bit a breathing space between it and the label.